src/Tickets/Seating/Admin/Tabs/Layout_Card.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * The Layout card object class.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Seating\Admin\Tabs;
+ */
+
+namespace TEC\Tickets\Seating\Admin\Tabs;
+
+/**
+ * The Layout_Card Class.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Seating\Admin\Tabs;
+ */
+class Layout_Card {
+	/**
+	 * The Layout ID.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected string $id;
+		
+	/**
+	 * The Layout name.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected string $name;
+	
+	/**
+	 * The Layout map ID.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected string $map;
+		
+	/**
+	 * The number of seats in the Layout.
+	 *
+	 * @since TBD
+	 *
+	 * @var int
+	 */
+	protected int $seats;
+		
+	/**
+	 * The URL to the Layout's screenshot.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected string $screenshot_url;
+		
+	/**
+	 * Layout_Card constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $id The Layout ID.
+	 * @param string $name The Layout name.
+	 * @param string $map The Layout map ID.
+	 * @param int    $seats The number of seats in the Layout.
+	 * @param string $screen_shot_url The URL to the Layout's screenshot.
+	 */
+	public function __construct( string $id, string $name, string $map, int $seats, string $screen_shot_url ) {
+		$this->id             = $id;
+		$this->name           = $name;
+		$this->map            = $map;
+		$this->seats          = $seats;
+		$this->screenshot_url = $screen_shot_url;
+	}
```

Comment:
I know is a bit silly, but would love to see a factory method, with the screenshot url being a setter method since it shouldn't be required, I wonder is the screenshot URL an external link or an image that we upload?